html2docx/html2docx.py

```diff
@@ -101,6 +104,36 @@ def finish_p(self) -> None:
             self.r.text = self.r.text.rstrip()
         self._reset()
 
+    def init_table(self, attrs: List[Tuple[str, Optional[str]]]) -> None:
+        container = self.doc if self.table_cell is None else self.table_cell
+        table = container.add_table(rows=0, cols=0)
+        self.tables.append((table, -1, -1))
+
+    def finish_table(self) -> None:
+        table = self.tables.pop()[0]
+        section = self.doc.sections[0]
+        page_width = section.page_width - section.left_margin - section.right_margin
+        page_width = int(page_width * (0.5 ** len(self.tables)))
```

Comment:
Can you please explain this line? I don't follow why the page width is multiplied by 0.5 ^ len(self.tables). Thanks.

Reply:
I can't. I did not wrote this line, this is from a previous pull request (https://github.com/erezlife/html2docx/pull/55), I just took the existing code and worked on the tests which was the main feedback.
If I had to guess: this is an estimation of the width of the table which is smaller the more nested is the table.

Reply:
Unfortunately the original reviewer is no longer with our organization. I can't approve this PR until I fully understand what this line does. If you like you can fork the repo and merge it in on your fork.

Reply:
OK I understand. This line is originally to handle nested table which I think is fairly rare. If you agree, I will just remove the "nested" feature so that we can start with this simple case and then we can add more edge cases in the a future pull request?

Reply:
Sure, that's a good idea